public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -43,7 +42,7 @@ export function TablePanel(props: Props) {
     <Table
       height={tableHeight}
       // This calculation is to accommodate the optionally rendered Row Numbers Column
-      width={options.showRowNums ? width : width + OPTIONAL_ROW_NUMBER_COLUMN_WIDTH}
```

Comment:
this logic is really confusing, how can showRowNums impact the total width of the Table componet??? like the space available for the Table can't change if this option is enabled or not. This logic would make sense if we showed the row number outside the Table component